### PR TITLE
Feature/pdct 1095 move document type to taxonomy

### PR DESCRIPTION
**Pull request by katybaulch**
# Description

- Additive change to move document type into _document taxonomy
- Bump db client to 3.8.7

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [x] Patch  
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] GitHub workflow update
- [ ] Documentation update
- [ ] Remove legacy code
- [ ] Dependency update

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [ ] The PR represents a single feature (small drive-by fixes are also ok)
- [ ] The PR includes tests that are sufficient for the level of risk
- [ ] The code is sufficiently commented, particularly in hard-to-understand areas
- [ ] Any required documentation updates have been made
- [ ] Any TODOs added are captured in future tickets
- [ ] No FIXMEs remain

### CODE REVIEW

File: tests/integration_tests/document/test_update.py [hunk 1]
```diff
@@ -58,8 +54,7 @@ def test_update_document_super(
     data = response.json()
     assert data["import_id"] == "D.0.0.2"
     assert data["variant_name"] == "Translation"
-    assert data["metadata"] == {"role": ["SUMMARY"]}
-    assert data["type"] == "Annex"
+    assert data["metadata"] == {"role": ["SUMMARY"], "type": ["Annex"]}
```
Comment (diversemix) [inline tests/integration_tests/document/test_update.py:57]: Mayne we can pullout a `DEFAULT_DOC_META` constant? Which would make subsequent changes a little easier?

**Review by diversemix — APPROVED** [review]
lgtm

File: tests/integration_tests/document/test_update.py (hunk 1 above)
Comment (katybaulch) [reply tests/integration_tests/document/test_update.py:57]: Will do this in the next PR :)

**Review by annaCPR — APPROVED** [review]
All good

**Pull request merged**